osm-fudge/__main__.py

```diff
@@ -0,0 +1,27 @@
+import bk_tree
+import sys
+
+import levenshtein
+import jaccard
+
+distance_metrics = {
+	'levenshtein': levenshtein.find_levenshtein_distance,
+	'jaccard': jaccard.find_jaccard_similarity
+}
+
+with open(sys.argv[1], 'r') as file:
+	initial_word = file.readline()
+	metrics = sys.argv[2]
+	if metrics not in distance_metrics.keys():
+		sys.exit("Allowed metrics are {metrics}".format(metrics=list(distance_metrics.keys())))
```

Comment:
Check out the argparse module in the Python stdlib; it comes with the functionality you implemented on your own here:

https://docs.python.org/3/library/argparse.html#choices

This allows you to add command line argument which can be restricted to specific values.

Reply:
Yes yes, that is a good library. Also helps in setting required argument.